src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Translating Files
+---
+
+Files uses Crowdin for translation management, which is a cloud-based SaaS that supports globalization of a variety of application platforms.
```

Comment:
- [ ] Translations for Files are done through the popular crowd based platform Crowdin. Anyone with a Crowdin account can translate strings and help improve existing strings.

Reply:
Done